### DISCUSSION TITLE
【大理】zkp 线下共学 @Demian

### DISCUSSION THREAD

**Discussion by Demian101** [zkp-co-learning/zkp-co-learn#6] [Co-learn IRL！]
zkp(零知识证明) 大理线下共学 Co-learn
即刻报名 且 大理在地的朋友，可来大理 706 参与线下 co-learn，

📌地址：大理 706
⏰时间：2.21 开始每周 2/4/6 晚 20:00 -22:00 
👬🏻人数：限 6-7 人

学习形式：自学 + 讨论
线下详情可联系 @Demian (wechat: soda1543)

Co-learn 内容： 706 zkp 课程资料
Co-learn 方法：共 2 轮，每轮包括50min 非依赖(闭麦)自学+ 10min Discussion

**Comment by Demian101** [comment]
Part 1 活动介绍
线下的 [共学Group] 有别于 [授课]课程，在地的 co-learn 会有这几个特点：
1. 社区性（聚在一起是在地社群 vibe 形成最方便的基础）
2. 有效性（集智学习 + 协同工作可以节约时间，将资料的效用最大化）
3. 能动性（克服拖延, 提高效率)

Why ZK？
…

Part 2 活动形式：
1. 每周 2/4/6两次集合式workshop 2H（推荐线下，线上腾讯会议同步）
2. workshop采用番茄时钟制， 每轮1h*2到3轮 , 分为 Independent study 45min , 10min discussion 和 5 min 休息
3. Independent study 强调自主性, 遇到问题先尝试自己搜索（chatgpt、Google、知乎）, 搜索后仍未解决的问题，需要统一放置到 Notion 的 DiscussionPage 页面，在10min 的 Discussion 环节由大家一起探讨 (电视投屏)

Notion DiscussionPage 提问形式：
Title：2022-01-23 ZK co-learn Discussion Page
提问人：@小明
问题描述：为什么....xxxx
搜索渠道：chatgpt、Google、知乎 仍未得到答案

Part3 参与门槛
1. Base 大理可线下
2. 报名 706×Antalpha zkp 共学课程

Part4 你可以收获
1. 志同道合的共学伙伴，未来的开发队友
2. 即兴趣味的项目共创
3. 积极的学习氛围、有趣的课后活动
4. 更多惊喜待你挖掘…

Par5 签到、每人进度更新
AMA

**Reply by ShirleneLiu** [reply]
来个群二维码？

**Reply by Demian101** [reply]
群二维码容易过期，直接加我：
soda1543
